### DISCUSSION TITLE
Enable SAML SSO on Free Organization

### DISCUSSION THREAD

**Discussion by kpap-equiti** [community/community#47648] [Enterprise]
Hello, 
I would like to ask if it is possible to enable SAML SSO on the free tier organization. 

Thank you

**Comment by queenofcorgis** [comment]
Hi there! 

Thanks for asking @kpap-equiti ⭐ . Currently, no - this [doc](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on) gives you more information as it is only available to Enterprise Cloud users. 

If you'd like to give product feedback, I recommend creating a post explaining your use case in the [General category](https://github.com/orgs/community/discussions/categories/general?discussions_q=sort%3Atop+created%3A%3E%3D2023-01-22+no%3Alabel+category%3AGeneral) :octocat: .
